test/resources/pr2_desc.3.srdf

```diff
@@ -69,4 +69,9 @@
    <disable_collisions link1="r_shoulder_pan_link" link2="l_gripper_palm_link" />
    <!-- and many more disable_collisions tags -->
 
-</robot>
\ No newline at end of file
+   <joint_property joint_name="world_joint" property_name="angular_distance_weight" value="0.5" />
+
+   <!-- When parsing, this made up joint that is not present in the URDF is expected to print a warning -->
```

Comment:
```suggestion
   <!-- When parsing, this made up joint that is not present in the URDF is expected to print an error -->
```

Reply:
But it does print a warning? 

     CONSOLE_BRIDGE_logWarn("Property defined for a joint '%s' that is not known to the URDF. Ignoring.",
                             jp.joint_name_.c_str());

Reply:
You are right, sorry. I was confused by all the logError messages a few lines above.
It feels somewhat inconsistent to use errors for some critical mistakes and warnings for others.
After all, all these failed conditions result in the definition being ignored.